Bottom: a779b5ff23d4b8d902838323658a3501cb799aa0
Top:    81f161bb2f2c3b930e36c572a75df598be1ba0dd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-02 22:26:30 +0000

Avoid clearing pins / vias from polygons... it is already done!



---

diff --git a/src/draw.c b/src/draw.c
index e944790..8983654 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1262,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1284,6 +1285,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
